static/base.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
```

Comment:
I think it's better to move it to `templates/index.html`:
- keeping the same filename make it easier to guess that it'll be outputed as `index.html` while `base.html` is less straightforward
- it'll be possible to remove the whole `templates` folder after build step so "templates" files are not served and will return a 404